killrvideo-commons/src/main/java/com/killrvideo/conf/DseDriverConfiguration.java

```diff
@@ -0,0 +1,302 @@
+package com.killrvideo.conf;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+/**
+ * The DSE (DataStax Enterprise) Driver configuration.
+ *
+ * <p>
+ * Driver options should be specified in the usual way, that is, through an <code>
+ * application.conf</code> file accessible on the application classpath. See the
+ * <a href="https://docs.datastax.com/en/developer/java-driver-dse/2.0/manual/core/configuration/">driver configuration</a>
+ * section in the online docs for more information.
+ *
+ * <p>
+ * To illustrate how to integrate the driver configuration with Spring, a few driver options should be configured through Spring's
+ * own configuration mechanism:
+ *
+ * <ol>
+ * <li><code>killrvideo.dse.contactPoints</code>: this property will override the driver's {@code
+ *       datastax-java-driver.basic.contact-points} option; it will default to <code>127.0.0.1
+ *       </code> if unspecified;
+ * <li><code>killrvideo.dse.port</code>: this property will be combined with the previous one to create initial contact points; it will
+ * default to <code>9042</code> if unspecified;
+ * <li><code>driver.localdc</code>: this property will override the driver's {@code
+ *       datastax-java-driver.basic.load-balancing-policy.local-datacenter} option; it has no default value and must be specified;
+ * <li><code>driver.keyspace</code>: this property will override the driver's {@code
+ *       datastax-java-driver.basic.session-keyspace} option; it has no default value and must be specified;
+ * <li><code>driver.consistency</code>: this property will override the driver's {@code
+ *       datastax-java-driver.basic.request.consistency}; it will default to <code>LOCAL_QUORUM
+ *       </code> if unspecified;
+ * <li><code>driver.pageSize</code>: this property will override the driver's {@code
+ *       datastax-java-driver.basic.request.page-size}; it will default to <code>10</code> if unspecified;
+ * <li><code>driver.username</code>: this property will override the driver's {@code
+ *       datastax-java-driver.advanced.auth-provider.username} option; if unspecified, it will be assumed that no authentication
+ * is required;
+ * <li><code>driver.password</code>: this property will override the driver's {@code
+ *       datastax-java-driver.advanced.auth-provider.password} option; if unspecified, it will be assumed that no authentication
+ * is required;
+ * </ol>
+ *
+ * The above properties should be typically declared in an {@code application.yml} file.
+ * 
+ * @author DataStax Developer Advocates team.
+ * @param <RegularImmutableSet>
+ */
+@Configuration
+@Profile("!unit-test & !integration-test")
+public class DseDriverConfiguration {
+
+//    /** Initialize dedicated connection to ETCD system. */
```

Comment:
Why is all of this commented out? Is it needed?

Reply:
This is not needed Anymore. I will completely delete it.